.github/mergeable.yml

```diff
@@ -0,0 +1,32 @@
+
+version: 2
+mergeable:
+  - when: pull_request.*, pull_request_review.*
+    validate:
+      - do: title
+        must_exclude:
+          regex: ^\[WIP\]
+          message: This is work in progress. Do not merge yet.
+        must_include:
+          regex: ^(feat|docs|chore|fix|test)(\(\w+\))?:.+$
+          message: Semantic release conventions must be followed.
+      - do: description
+        must_exclude:
+          regex: \[ \]
+          message: There are incomplete TODO task(s) unchecked.
+      - do: approvals
+        min:
+          count: 2
```

Comment:
Can we change this to 1? The `or` doesn't seem to be picking up Inna's approval.